code/xcvm/evm/src/interfaces/IInterpreter.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.14;
```

Comment:
Either we should have `pragma solidity 0.8.14` or `pragma solidiy ^0.8.14` I guess.